pkgs/applications/version-management/yadm/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchurl, git, bash }:
```

Comment:
They are necessary for yadm to execute, and I thought that was how to specify dependencies.